src/components/Table/GridTable.tsx

```diff
@@ -730,12 +727,35 @@ const VirtualRoot = memoizeOne<
         css={{
           // Add an extra `> div` due to Item + itemContent both having divs
           ...Css.addIn("& > div + div > div > *", gs.betweenRowsCss || {}).$,
-          // Table list styles only
-          ...(isHeader
-            ? {}
-            : {
-                ...Css.addIn("& > div:first-of-type > *", gs.firstNonHeaderRowCss).$,
-              }),
+          // Table list styles
+          ...Css.addIn("& > div:first-of-type > *", gs.firstNonHeaderRowCss).$,
+          ...gs.rootCss,
+          ...(gs.minWidthPx ? Css.mwPx(gs.minWidthPx).$ : {}),
+          ...xss,
+        }}
+        data-testid={id}
+      >
+        {children}
+      </div>
+    );
+  });
+});
+
+const VirtualHeader = memoizeOne<
```

Comment:
@stephenh - I stole this `memoizeOne` from the `VirtualRoot` definition, though I am wondering if this is necessary here, or even still necessary in the VirtualRoot? Seems like it was to help with Virtuosos rendering.. but not sure I completely follow what's going on here.

Reply:
Fwiw if they had two props, like topItemComponent and topItemProps where we could pass in extra props to our header/list components we wouldn't need this, but I don't think they have those?

Reply:
No, they just have you pass your props in directly...
```
components={{
  TopItemList: React.forwardRef((props, ref) => <CustomComponent customProp={prop} {...props} ref={ref} />);
}}
```

Reply:
...huh, and that doesn't break their memorization? I mean, if it doesn't and I was just making that up, then 🤷 , yeah sounds like we could remove the memoizeOnes.

Reply:
haha. Do you recall what was breaking memoization? Was it on filtering, scrolling, etc... ?

Reply:
I am testing it out now by just scrolling and see how it memoizes/rerenders. Will report back soon.

Reply:
@stephenh - When scrolling the table to force the VirtualRoot to rerender, I am not really seeing a difference between using the `memoizeOne` and not using it.
I think for now I'm going to keep it as is. If we want to change the I'd prefer to do it in an isolated PR in the event it really breaks something and we have to revert.
Though, I have made some changes to the `TopItemList` component, because turns out it is just for the wrapping element. `TopItemList` just wraps the `List`/`VirutalRoot` component, so it is much simpler than I originally made it.